pkg/discovery/dtofactory/container_spec_entity_dto_builder.go

```diff
@@ -169,7 +174,11 @@ func aggregateThrottlingSamples(samples [][]metrics.ThrottlingCumulative) (float
 	}
 	avgThrottledOverall := float64(0)
 	if totalOverall > 0 {
-		avgThrottledOverall = throttledOverall * 100 / totalOverall
+		if throttledOverall > totalOverall {
+			avgThrottledOverall = 100
```

Comment:
Thanks for addressing this, can we log a warning message here?

Reply:
Done!